mmdet/integration/nncf/compression.py

```diff
@@ -61,6 +64,30 @@ def is_checkpoint_nncf(path):
     except FileNotFoundError:
         return False
 
+def get_nncf_config_from_meta(path, log_dir=None):
```

Comment:
I guess it is worth removing `log_dir` from here

Reply:
I think it should be a good idea in the future, so I'd like to keep it. 
But if your opinion is to remove it definitely, I will remove it

Reply:
don't care about future so much, it is not used for now, let's remove it

Reply:
Ok

Reply:
Fixed